src/components/blocks/review-form/review-form.tsx

```diff
@@ -1,19 +1,42 @@
-import {ChangeEvent, Fragment, useState} from 'react';
+import {ChangeEvent, FormEvent, Fragment, useRef, useState} from 'react';
 import {RATINGS, ReviewLength} from './const.tsx';
+import {useAppDispatch, useAppSelector} from '../../../hooks';
+import {getIsReviewLoading} from '../../../store/offer-data/offer-data.selectors.ts';
+import {postReviewAction} from '@/store/user/user.api-actions.ts';
 
-function ReviewForm(): JSX.Element {
-  const [review, setReview] = useState({rating: 0, review: ''});
+function ReviewForm({offerId}: {offerId: string}): JSX.Element {
+  const initialState = {rating: 0, review: ''};
+  const [review, setReview] = useState(initialState);
+  const reviewRef = useRef<HTMLTextAreaElement | null>(null);
+  const ratingRef = useRef<HTMLInputElement | null>(null);
+  const dispatch = useAppDispatch();
+  const isReviewLoading = useAppSelector(getIsReviewLoading);
   const handleChange = (event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
```

Comment:
Не сжимай код, дай ему "дышать") отделяй пустой строкой логические блоки и так же всякие обработчики и тд и тп 
```suggestion
  const isReviewLoading = useAppSelector(getIsReviewLoading);
  
  const handleChange = (event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
```